packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -138,25 +131,22 @@ export function InitializeBindings() {
 
     component.componentWillLoad = function () {
       const element = getElement(this);
-      const event = buildCustomEvent(
-        initializeEventName,
-        (bindings: Bindings) => {
-          this.bindings = bindings;
-
-          const updateLanguage = () => forceUpdate(this);
-          this.bindings.i18n.on('languageChanged', updateLanguage);
-          unsubscribeLanguage = () =>
-            this.bindings.i18n.off('languageChanged', updateLanguage);
-
-          try {
-            // When no controller is initialized, updating a property with a State() decorator, there will be no re-render.
-            // In this case, we have to manually trigger it.
-            this.initialize ? this.initialize() : forceUpdate(this);
-          } catch (e) {
-            this.error = e as Error;
-          }
+      const event = buildCustomEvent(initializeEventName, (bindings: any) => {
```

Comment:
eslint ignore here too or another type

Reply:
Instead of `any`, could we use `AnyBindings`?

Reply:
Yeah, those any are not good. I want to tackle more refactoring of the types here as part of a separate PR, however.

This PR is starting to become very large as it is. I'll tackle some temp eslint rules.